packages/dds/tree/src/feature-libraries/forestIndex.ts

```diff
@@ -165,9 +164,9 @@ export class ForestIndex implements Index<unknown>, SummaryElement {
         const placeholderTree = JSON.parse(decodedtree) as PlaceholderTree[];
 
         // TODO: maybe assert forest is empty?
-        const range = this.forest.add(placeholderTree.map((t) => new TextCursor(t)));
-        const dst = { index: 0, range: this.forest.rootField };
-        this.forest.attachRangeOfChildren(dst, range);
+        // TODO: is this the correct delta?
+        const insert: Delta.Insert = { type: Delta.MarkType.Insert, content: placeholderTree };
```

Comment:
It is! :)
At least until we refactor everything to have a real root.
```suggestion
        const insert: Delta.Insert = { type: Delta.MarkType.Insert, content: placeholderTree };
```